packages/api-client-core/package.json

```diff
@@ -40,6 +40,7 @@
     "tiny-graphql-query-compiler": "^0.2.2",
     "tslib": "^2.6.2",
     "type-fest": "^3.3.0",
+    "wonka": "^6.3.2",
```

Comment:
I'm guessing we always had this transitively so it doesn't affect bundle size?

Reply:
Correct!